libbpf-rs/src/map.rs

```diff
@@ -1203,40 +1213,112 @@ bitflags! {
 #[non_exhaustive]
 #[repr(u32)]
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
-// TODO: Document members.
-#[allow(missing_docs)]
 pub enum MapType {
+    /// An unspecified map type.
     Unspec = libbpf_sys::BPF_MAP_TYPE_UNSPEC,
+    /// A general purpose Hash map storage type.
+    /// Refer the kernel [documentation](https://docs.kernel.org/bpf/map_hash.html) for more details.
     Hash = libbpf_sys::BPF_MAP_TYPE_HASH,
+    /// An Array map storage type.
+    /// Refer the kernel [documentation](https://docs.kernel.org/bpf/map_array.html) for more details.
     Array = libbpf_sys::BPF_MAP_TYPE_ARRAY,
+    /// A program array map which holds only the file descriptors to other eBPF programs. Used for
+    /// tail-calls.
     ProgArray = libbpf_sys::BPF_MAP_TYPE_PROG_ARRAY,
+    /// An array map which holds only the file descriptors to perf events.
     PerfEventArray = libbpf_sys::BPF_MAP_TYPE_PERF_EVENT_ARRAY,
+    /// A Hash map with per CPU storage.
+    /// Refer the kernel [documentation](https://docs.kernel.org/bpf/map_hash.html#per-cpu-hashes) for more details.
```

Comment:
```suggestion
    /// A Hash map with per CPU storage.
    ///
    /// Refer the kernel [documentation](https://docs.kernel.org/bpf/map_hash.html#per-cpu-hashes) for more details.
```

Reply:
Let's add an empty line between the summary and details, as is rustdoc standard from what I understand? Same below.